README.md

```diff
@@ -90,6 +90,20 @@ You can have a look at [the documentation](./docs/api.md).
 
 You should have a look at [the pitfalls](./docs/pitfalls.md).
 
+# Accessibility support
+
+For now, accessibility support is experimental.
+Since we bypass the native focus, and the screen readers rely on the native elements, it's
+a difficult topic.
+
+We export a hook that provides you props to provide to your focusable elements.
```

Comment:
It seems 'heavy' to repeat the verb "provide" here, maybe you could replace the first occurence with "returns" or the second one with "passes"